.golangci.yml

```diff
@@ -97,11 +97,11 @@ linters:
     disable-all: true
     enable:
         ## enabled by default
-        - errcheck # checking for unchecked errors, these unchecked errors can be critical bugs in some cases
+        #- errcheck # checking for unchecked errors, these unchecked errors can be critical bugs in some cases
         - gosimple # specializes in simplifying a code
-        - govet # reports suspicious constructs, such as Printf calls whose arguments do not align with the format string
+        #- govet # reports suspicious constructs, such as Printf calls whose arguments do not align with the format string
```

Comment:
I would argue that `govet` and `errcheck` are important to avoid rather fundamental bugs, do they still report issues?